readme-vars.yml

```diff
@@ -38,10 +38,10 @@ param_env_vars:
 opt_param_usage_include_env: true
 opt_param_env_vars:
   - { env_var: "JELLYFIN_PublishedServerUrl", env_value: "192.168.0.5", desc: "Set the autodiscovery response domain or IP address."}
-opt_param_usage_include_vols: true
+opt_param_usage_include_vols: false
 opt_param_volumes:
   - { vol_path: "/opt/vc/lib", vol_host_path: "/opt/vc/lib", desc: "Path for Raspberry Pi OpenMAX libs *optional*." }
-opt_param_device_map: true
+opt_param_device_map: false
```

Comment:
If the optional devices section is being disabled should we also remove the optional devices entries?

Reply:
I decided to keep it, in case we want to go the other way in the future